### DISCUSSION TITLE
How to change commands

### DISCUSSION THREAD

**Discussion by ElCunq** [ahihiyou20/discord-selfbot-owo-bot#16] [Q&A] [answered]
Somehow owo's /hunt and /battle commands crashing after a while. So I want to prevent that with just change / command with owo. Is this possible?

**Comment by ahihiyou20** [comment, marked as answer]
@ElCunq I didn't added that feature. But it will be added in the next update!

**Reply by ElCunq** [reply]
hey I got another question. It won't start again after solving the capcha. Its only closing. can you help about it?

**Reply by ahihiyou20** [reply]
> hey I got another question. It won't start again after solving the capcha. Its only closing. can you help about it?

It should do it after solving the captcha. However, there can be an issue prevent it from restarting. You can fix this by rename your folders. Make sure there's no spaces or special characters

**Reply by ElCunq** [reply]
So I did what u said and renamed the file "owobot" but app got broken after it. App only shows blank console. I tried to delete the whole files and take them back from the zip and it is still broken.

![resim_2022-12-01_163840323](https://user-images.githubusercontent.com/112334725/205067168-8641ba66-12c5-4635-88a2-2350df7c2140.png)

**Reply by ElCunq** [reply]
> So I did what u said and renamed the file "owobot" but app got broken after it. App only shows blank console. I tried to delete the whole files and take them back from the zip and it is still broken.
> 
> ![resim_2022-12-01_163840323](https://user-images.githubusercontent.com/112334725/205067168-8641ba66-12c5-4635-88a2-2350df7c2140.png)

somehow it got fixed by itself. So I don't need help I guess.
